hyper/hyper/restore_test.go

```diff
@@ -0,0 +1,57 @@
+package hyper_test
+
+import (
+	"testing"
+
+	"github.com/isard-vdi/isard/hyper/hyper"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRestore(t *testing.T) {
+	require := require.New(t)
+	assert := assert.New(t)
+
+	cases := map[string]struct {
+		PrepareDesktop func(h *hyper.Hyper) string
+		ExpectedErr    string
+	}{
+		"restore the desktop correctly": {
+			PrepareDesktop: func(h *hyper.Hyper) string {
+				desktop, err := h.Start(hyper.TestMinDesktopXML(t), &hyper.StartOptions{})
+				require.NoError(err)
+
+				err = h.Save(desktop, "test.dump")
+				require.NoError(err)
+
+				return "test.dump"
+			},
+		},
+		"should return an error if there's an error restoring the desktop": {
+			PrepareDesktop: func(h *hyper.Hyper) string {
+				return ""
+			},
+			ExpectedErr: "virError(Code=38, Domain=12, Message='incomplete save header in '/home/darta/github/nouisard/hyper/hyper/': Is a directory')",
```

Comment:
This test would fail in other machines than yours. Try making it fail in another way (see [list_test.go](https://github.com/NefixEstrada/isard/blob/e8f1fc326b686030fe2440bcf00a70bfdaff6f7d/hyper/hyper/list_test.go#L24) for example)

Reply:
Fixed